.github/workflows/build-test.yml

```diff
@@ -4,7 +4,14 @@ on: [push]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runner }}
+
+    strategy:
+      matrix:
+        runner:
+          - windows-2022
+          - ubuntu-20.04
+          - macos-11
```

Comment:
(Hmm, I wonder if we really need to _build_ on all supported platforms like this. There is probably a bit of redundancy in that we test the `publish-packages` job - on Linux - anyway, so maybe we could leave this out altogether. Will think about it, it will let us skip some of the CI jobs if we make that change.)

Reply:
Removed the `build` job. We'll let `publish-packages` take care of this and minimize the duplication this way.